src/java.base/share/classes/jdk/internal/loader/NativeLibraries.java

```diff
@@ -384,6 +384,20 @@ boolean open() {
         }
     }
 
+    public static NativeLibrary defaultLibrary = new NativeLibraryImpl(Object.class, "<default>", true, true) {
```

Comment:
Spotbugs reported this field can be final -

MS: In class jdk.internal.loader.NativeLibraries
In class jdk.internal.loader.NativeLibraries
Field jdk.internal.loader.NativeLibraries.defaultLibrary
At NativeLibraries.java:[line 387]
jdk.internal.loader.NativeLibraries.defaultLibrary isn't final but should be